### DISCUSSION TITLE
please tell me your favorite anime!

### DISCUSSION THREAD

**Discussion by MGMCN** [NAIST-SE/AttractingContributors#403] [General]
https://github.com/MGMCN/AttractingContributors_jp_anime

**Comment by AtropsCooper** [comment]
Anime added.
Help me please if you'd like to 🥺
https://github.com/AtropsCooper/AttractingContributors-Random

**Comment by aru1702** [comment]
I think it can be more helpful if you added some specification.
For example, action genre, shounen anime, etc.
I will add mine, sharing is caring, and good luck!

https://github.com/aru1702/AttractingContributors-TouhouFumo

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter
